### DISCUSSION TITLE
PGlite running Postgres in Browser with FerretDB?

### DISCUSSION THREAD

**Discussion by al-sabr** [FerretDB/FerretDB#4614] [General]
![image](https://github.com/user-attachments/assets/393118bf-de66-4feb-a09e-25f787105ab1)

Is it possible to make something compatible with this project that allows Postgres to run inside of the browser.

https://github.com/electric-sql/pglite

https://supabase.com/blog/postgres-new

**Comment by AlekSi** [comment]
Yes, it should be possible. We are not likely to work on something like that, but we could help if someone in the community is interested.
